Resources/ServerInfo/_NF/Guidebook/Rules/9_Pirates.xml

```diff
@@ -3,11 +3,19 @@
 
 Pirates must follow their code.
 
+    - [color=#ff0000]The safe zone rules still apply to pirates.[/color]
     - Pirate crews may never exceed a total force of four members, including the Captain, sentient pets and cyborgs.
     -- Round start pets (e.g. Clarrrpy) do not count towards this limit.
     - There may only be one Pirate Captain and one First Mate at any time.
+    - Pirates may not operate without a First Mate or Captain present.
+    -- If the leader of the pirates has been captured or killed, the remaining pirates may make efforts to rescue them.
+    -- First Mates and Captains are required to open their job slot prior to going to cryo, or ahelp if this is not possible. Inform your crew of what they can do in your absence.
     - As a pirate, you must have LOOC permission before kidnapping and holding any civilian hostage.
     -- People kept hostage cannot be forced to do labor under any circumstance.
+    - If a player surrenders, the plundering of their vessel must be kept to a reasonable level.
+    -- Following escalation guidelines, if a player fights, then all goods on their ship are fair game. However, you must leave the ship in a state where it is able to fly (do not remove thrusters, power generation, shuttle console, crew console and all fuel).
+    -- If you need to escape, then a player requiring medical attention can be left on a point of interest as a last resort. Do your best to keep them as far from death beforehand.
+    -- If a player resumes combat after being revived, this can be taken as them accepting that they may have to respawn. In this case, they can be left to rot on a point of interest or left behind if saving them would compromise the safety of the pirate crew.
```

Comment:
This is really wordy.  Can it be boiled down?

Also, if you need to escape, that implies that a hostile force is there, and in that case, surely the duty of care for the attacked ship itself now falls unto them as the responder (NFSD or mercs) - your duty is to your crew first and foremost.

1. If an attacked ship surrenders, plunder is limited to X.  If not, plunder is unrestricted, but ships must be left in operational condition if possible.
2. If an attacked ship is not in operational condition, the crew must be assisted to a POI or friendly ship.
3. If enemy crewmates continue combat after being revived, they are considered "do not revive" and may be left for dead or stranded.
4. If you need to escape, any enemy crew _onboard_ must be revived and brought to a safe point when possible.

Reply:
"Also, if you need to escape, that implies that a hostile force is there, and in that case, surely the duty of care for the attacked ship itself now falls unto them as the responder (NFSD or mercs) - your duty is to your crew first and foremost."
As per the escalation rules: You have to seek medical attention for the person you've killed. - This is written in line with that. 

Can see if I can water it down a bit otherwise.

Reply:
We have to try and maintain consistent care for players to avoid them being round removed. We have to encourage the pirates to avoid doing that somehow. Every admin has seen incidents where a player is killed while EVA by pirates and ends up endlessly drifting. 
Also, honestly, if you're pirating with the NFSD that close to you, you're doing it wrong.